src/pages/all-on-one-v2/all-on-one-v2.ts

```diff
@@ -109,8 +125,19 @@ export class AllOnOneV2Page {
     return this.selectedManoeuvre === key;
   }
 
+  controlledStopTap() {
+    this.isControlledStopDone = true;
```

Comment:
There's no way to reverse this. Is that right?

Reply:
Yeah, it's doable but a bit of a nightmare. It conflicts with the undo'ing of serious/dangerous faults as they want to use the (click) event as well.

Reply:
Think the PO's are ok with it to be honest, I'll double check with them

Reply:
I think it is fine for now - there's a fair bit of inconsistency all over the DL25 and I think it all needs a bit of a holistic rethink

Reply:
Couldn't agree with you more, rethink and a rewrite!